### DISCUSSION TITLE
新更新的2.11.4不能调节亮度

### DISCUSSION THREAD

**Discussion by ContinueStudio** [shuax/MouseInc.Settings#87] [Q&A]
一直显示当前亮度是100%，调节声音都是正常的

**Comment by shuax** [comment]
你的电脑在设置能调节亮度吗。然后你在powershell执行这个看看结果。
`(Get-WmiObject -ns root/wmi -class wmiMonitorBrightness).CurrentBrightness`

**Reply by ContinueStudio** [reply]
电脑上还可以调节亮度吗？我用这个插件之前都是手动在显示器上调的。。用了才这个才发现原来世界如此美妙。。但是不明白，为什么要改呀，之前那个调节我觉得很好用阿

**Comment by ContinueStudio** [comment]
PS C:\WINDOWS\system32&gt; (Get-WmiObject -ns root/wmi -class wmiMonitorBrightness).CurrentBrightness
Get-WmiObject : 不支持
所在位置 行:1 字符: 2
+ (Get-WmiObject -ns root/wmi -class wmiMonitorBrightness).CurrentBrigh ...
+&nbsp; ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
&nbsp; &nbsp; + CategoryInfo&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; : InvalidOperation: (:) [Get-WmiObject]，ManagementException
&nbsp; &nbsp; + FullyQualifiedErrorId : GetWMIManagementException,Microsoft.PowerShell.Commands.GetWmiObjectCommand





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "shuax/MouseInc.Settings"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月5日(星期四) 下午5:52
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shuax/MouseInc.Settings] 新更新的2.11.4不能调节亮度 (#87)





 
你的电脑在设置能调节亮度吗。然后你在powershell执行这个看看结果。
 (Get-WmiObject -ns root/wmi -class wmiMonitorBrightness).CurrentBrightness
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by shuax** [comment]
`(Get-Ciminstance -Namespace root/wmi -class wmiMonitorBrightness).CurrentBrightness`
再试试这个

**Reply by ContinueStudio** [reply]
PS C:\WINDOWS\system32> (Get-Ciminstance -Namespace root/wmi -class wmiMonitorBrightness).CurrentBrightness
Get-Ciminstance : 不支持
所在位置 行:1 字符: 2
+ (Get-Ciminstance -Namespace root/wmi -class wmiMonitorBrightness).Cur ...
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotImplemented: (root/wmi:wmiMonitorBrightness:String) [Get-CimInstance], CimException
    + FullyQualifiedErrorId : HRESULT 0x8004100c,Microsoft.Management.Infrastructure.CimCmdlets.GetCimInstanceCommand

**Comment by shuax** [comment]
了解了，下个版本修复。

**Reply by seldoms** [reply]
跪谢大佬，大佬辛苦了!
